### DISCUSSION TITLE
The channel was closed: AMQP close-reason, initiated by Peer, code=503

### DISCUSSION THREAD

**Discussion by attiqeurrehman** [rabbitmq/rabbitmq-server#10864] [Questions]
Hi,

We are facing the following issue:

> System.InvalidOperationException: The channel was closed: AMQP close-reason, initiated by Peer, code=503, text='COMMAND_INVALID - unknown exchange type 'x-delayed-message'', classId=40, methodId=10 9

I understand from previous discussions that this error can occur when the [rabbitmq-delayed-message-exchange](https://github.com/rabbitmq/rabbitmq-delayed-message-exchange) plugin is not enabled. However, in our case, we've confirmed the plugin is enabled, yet we're still encountering the issue.  Do you have any suggestions on what else might be causing the problem?

**Comment by lukebakken** [comment]
Please don't cross-post.

I will reply here - https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/discussions/275
